app/assets/stylesheets/products.scss

```diff
@@ -0,0 +1,167 @@
+// Place all the styles related to the products controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.slider {
+  height: 540px !important;
+}
+.slides {
+  height: 500px !important;
+  min-height: 500px !important;
+  // width: 500px !important;
+  li, li.active {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own single line